test/driver_stub/cl.c

```diff
@@ -902,6 +902,7 @@ clCompileProgram(cl_program            program ,
                  void (CL_CALLBACK *   pfn_notify)(cl_program  program , void *  user_data),
                  void *                user_data) CL_API_SUFFIX__VERSION_1_2
 {
+    (void)input_headers;
```

Comment:
I have a slight preference to log `input_headers` here and in the test itself vs. marking it as unused, but I understand if you don't want to make this change right now.

Reply:
I had the same though as you at first, but then I started wondering if the ignored parameters were not dynamically allocated buffers that would add randomness into the trace and render the comparison to the reference difficult. I agree it would be nice checking them as well, but it may require some changes to the infrastructure.